models/repo.go

```diff
@@ -961,12 +961,13 @@ func DeleteRepository(doer *user_model.User, uid, repoID int64) error {
 }
 
 type repoChecker struct {
-	querySQL, correctSQL string
-	desc                 string
+	querySQL   string
+	correctSQL func(e db.Engine, id int64) error
+	desc       string
 }
 
-func repoStatsCheck(ctx context.Context, checker *repoChecker) {
-	results, err := db.GetEngine(db.DefaultContext).Query(checker.querySQL)
+func repoStatsCheck(ctx context.Context, e db.Engine, checker *repoChecker) {
```

Comment:
No, you should use `context.Context` instead of `db.Engine`

Reply:
Thanks for the quick review! I made the changes to use context.Context as a function argument instead of db.Engine.

---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/34#note_808)